exportToBlogpost.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/bash
```

Comment:
Isn't this specific to your workflow? if yes, could you remove this file?

Reply:
Oops, you are right.